### PR TITLE
remove r channel from testing

### PR DESCRIPTION
**Pull request by bgruening**

### CODE REVIEW

**Comment by bgruening** [comment]
@daler @johanneskoester this is also ready for review.

**Comment by daler** [comment]
I think you got them all . . . what an annoying thing to grep for!

**Comment by bgruening** [comment]
Sorry was getting food, what happend :) Is it working? Yes, please let's wait an other day. The branch does not need to be empty but at least the strange ICU and readline issues needs to go away.

**Comment by bgruening** [comment]
You also had a similar branch? Btw. there is also `meta.build_number()` to return the build number.

**Comment by johanneskoester** [comment]
Yes, I basically had exactly the same change as you. What I meant was: this branch (kill_r) should not be merged into master before bulk is empty, right? Otherwise, we will run into problems I guess?

**Comment by bgruening** [comment]
Realistically, we need many more days for R. There are really broken recipes. At some point we put all on a blacklist and then we can merge this branch here. Work in a new bulk branch to get rid of the blacklist.

**Comment by johanneskoester** [comment]
Ok, but no need for a new bulk branch. You can simply merge bulk into master at some point (together with the blacklist), and go on in the very same bulk branch (the name is hardcoded and it is protected and so on, so it shall not be deleted).

**Comment by johanneskoester** [comment]
Mhm, after some more thoughts I think we cannot do that. When we blacklist R recipes and move to the new channel order in the master branch, all blacklisted recipes will be broken when people try to install them (e.g. because they are 3.3.1 and there is no r-base 3.3.1 in conda-forge). So, I think the bulk rebuild has to be complete before we merge the bulk branch and this PR into master.

**Comment by bgruening** [comment]
The bulk branch is "complete" there are a few packages blacklisted, but this was true before.
However, I think we can not disable the r-channel completely. As we do not store old versions in git these old versions are not rebuild and this means if people don't activate the r-channel they will not be able to install older version.

What do you think about adding the r-channel as last channel in the priority.

**Comment by johanneskoester** [comment]
Yes, we can do that for a while.
Great news! Thanks for working on this!!

**Comment by daler** [comment]
Awesome work Bjoern!

**Comment by bgruening** [comment]
@daler @johanneskoester ready when you are ready!

File: docs/source/contribute-a-recipe.rst [hunk 1]
```diff
@@ -144,7 +144,6 @@ You should recommend to your users that they set up the same channel order as
 used by bioconda to ensure that all dependencies are correctly met, by doing
 the following::
 
-    conda config --add channels conda-forge
     conda config --add channels defaults
```
Comment (johanneskoester) [inline docs/source/contribute-a-recipe.rst]: Above line can be removed now.

**Comment by johanneskoester** [comment]
Really nice!

**Pull request merged**